src/Orchard.Cms.Web/Modules/Orchard.Commons/Orchard.Commons.csproj

```diff
@@ -1,16 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\..\build\common.props" />
```

Comment:
There is a way to remove it by having the build.props file at the root of the folder.
Right @jtkech ? Also we need to ensure what is being done in the packages branch will not interfere.